WizardStyle.css

```diff
@@ -41,3 +41,11 @@ details {
 summary, .setAll {
   background-color: #F56600;
 }
+
+li {
+    margin-bottom: 2ex;
+}
+
+#vbutton {
```

Comment:
I cannot find the element with ID vbutton. Did you mean to add this to the CSS?

Reply:
At the time i added it I did, but got overtaken by events ... that bit can be removed.